packages/plugin-ext/src/main/browser/view/plugin-view-registry.ts

```diff
@@ -91,7 +91,8 @@ export class PluginViewRegistry implements FrontendApplicationContribution {
     private readonly viewsWelcome = new Map<string, ViewWelcome[]>();
     private readonly viewContainers = new Map<string, [string, ViewContainerTitleOptions]>();
     private readonly containerViews = new Map<string, string[]>();
-    private readonly viewClauseContexts = new Map<string, Set<string>>();
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    private readonly viewClauseContexts = new Map<string, any>();
```

Comment:
Does this need to be `<string, any>`? It seems that the options are `Set<string> | string` and maybe `| undefined`?